imagepicker/src/main/java/com/esafirm/imagepicker/features/ImageFileLoader.java

```diff
@@ -53,19 +53,31 @@ private ExecutorService getExecutorService() {
     private class ImageLoadRunnable implements Runnable {
 
         private boolean isFolderMode;
+        private boolean includeVideo;
         private ArrayList<File> exlucedImages;
         private ImageLoaderListener listener;
 
-        public ImageLoadRunnable(boolean isFolderMode, ArrayList<File> excludedImages, ImageLoaderListener listener) {
+        public ImageLoadRunnable(boolean isFolderMode, boolean includeVideo, ArrayList<File> excludedImages, ImageLoaderListener listener) {
             this.isFolderMode = isFolderMode;
+            this.includeVideo = includeVideo;
             this.exlucedImages = excludedImages;
             this.listener = listener;
         }
 
         @Override
         public void run() {
-            Cursor cursor = context.getContentResolver().query(MediaStore.Images.Media.EXTERNAL_CONTENT_URI, projection,
-                    null, null, MediaStore.Images.Media.DATE_ADDED);
+            // Return only video and image metadata.
+            String selection = MediaStore.Files.FileColumns.MEDIA_TYPE + "="
+                    + MediaStore.Files.FileColumns.MEDIA_TYPE_IMAGE;
+
+            if (includeVideo) {
+                selection = selection + " OR "
+                        + MediaStore.Files.FileColumns.MEDIA_TYPE + "="
+                        + MediaStore.Files.FileColumns.MEDIA_TYPE_VIDEO;
+            }
+
+            Cursor cursor = context.getContentResolver().query(MediaStore.Files.getContentUri("external"), projection,
```

Comment:
Instead of `MediaStore.Files.getContentUri("external")` i think we should still use `MediaStore.Images.Media.EXTERNAL_CONTENT_URI` for image only.